homeassistant/components/xiaomi_miio/switch.py

```diff
@@ -201,12 +201,20 @@
 
 
 @dataclass
-class XiaomiMiioSwitchDescription(SwitchEntityDescription):
+class XiaomiMiioSwitchRequiredKeyMixin:
+    """A class that describes switch entities."""
+
+    feature: int
+    method_on: str
+    method_off: str
```

Comment:
In order to mark these three as compulsory, I had to move them to a Mixin dataclass.
That removes the need to assert them in the code.